library/fluid/fluid_extensions/src/main/java/org/quiltmc/qsl/fluid/impl/QuiltFluidApiConstants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.fluid.impl;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.tag.TagKey;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+public class QuiltFluidApiConstants {
+
+	public static final TagKey<Fluid> QUILT_FLUIDS =
```

Comment:
Why is this needed?

Reply:
I was wondering the same, atleast maybe move it into lava-like and water-like maybe?

Reply:
I need this tag specifically in my entity mixins, to figure out if the player is swimming in a custom fluid, so i can enable swimming an other features, that would be hardcoded.

Reply:
Why does this need a tag, though? Can't you just do a `instance.getFluid() instanceof FlowableFluidExtensions` check? You're redirecting all the `isIn` checks anyway.

Reply:
big dumb dumb. ima do this real quick. pretend this never happened.